tests/test_disabled_root.h

```diff
@@ -0,0 +1,43 @@
+/**
+ * test_disabled_root.h
+ * =============================================================================
+ * Copyright (c) 2023-present Serhii Snitsaruk and the LimboAI contributors.
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ * =============================================================================
+ */
+
+#ifndef TEST_DISABLED_ROOT_H
+#define TEST_DISABLED_ROOT_H
+
+#include "limbo_test.h"
+
+#include "modules/limboai/blackboard/blackboard.h"
+#include "modules/limboai/bt/behavior_tree.h"
+#include "modules/limboai/bt/bt_instance.h"
+
+namespace TestDisabledRoot {
+
+TEST_CASE("[Modules][LimboAI] BTInstance when root task disabled FAILURE") {
+	Ref<BehaviorTree> bt = memnew(BehaviorTree);
+	Ref<BTTestAction> task = memnew(BTTestAction(BTTask::SUCCESS));
+	Node *dummy = memnew(Node);
+
+	SceneTree::get_singleton()->get_root()->add_child(dummy);
+	task->set_enabled(false);
+	bt->set_root_task(task);
+	Ref<Blackboard> blackboard = memnew(Blackboard);
+	Ref<BTInstance> bti = bt->instantiate(dummy, blackboard, dummy);
```

Comment:
```suggestion
	Ref<BTInstance> bti = bt->instantiate(dummy, blackboard, dummy, dummy);
```
We need to either pass a dummy as custom scene root here, or call `set_owner()` on the dummy with a valid node.